.github/workflows/main.yml

```diff
@@ -0,0 +1,31 @@
+name: tests
+on:
+  push:
+    branches:
+    - master
+    - dev
+    tags:
+    - v*
+  pull_request:
+    branches:
+    - dev
+    - master
+jobs:
+  test:
+    runs-on: 'ubuntu-latest'
+    timeout-minutes: 40  # default is 360
+    strategy:
+      matrix:
+        node-version: [8.x]
```

Comment:
We currently fix the version only in v8.9.0 because some compatibility issues between Ledger and Electron packages. We already have an issue to handle this upgrade: https://github.com/HathorNetwork/hathor-wallet/issues/196 but it must be done carefully